frontend/.gitignore

```diff
@@ -8,9 +8,6 @@
 # testing
 /coverage
 
-# production
-/build
-
```

Comment:
Django needs these files, since we are using the React app instead of a template.